docs/hub/_toctree.yml

```diff
@@ -57,6 +57,8 @@
     sections:
       - local: transformers
         title: 🤗 transformers
+      - local: diffusers
+        title: 🧨 diffusers
```

Comment:
not sure about the emoji here, i would remove it

```suggestion
        title: Diffusers
```

Reply:
(i would also remove it from Transformers in another PR, tbh)

Also can you keep the alphabetical order?

Reply:
(it was alphabetical with the emojis hehe)